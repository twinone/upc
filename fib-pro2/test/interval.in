+ 20.04.15 23:59 bad
+ 21.04.15 00:00 good
+ 22.04.15 23:59 good
+ 23.04.15 00:00 bad
? 21.04.15 22.04.15
sortir

# Author: Twinone
# Tests the limits of an interval query
