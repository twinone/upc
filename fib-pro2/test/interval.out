
+ 20.04.15 23:59 bad

+ 21.04.15 00:00 good

+ 22.04.15 23:59 good

+ 23.04.15 00:00 bad

? 21.04.15 22.04.15
1 good 21.04.15 00:00
2 good 22.04.15 23:59

sortir
